filebeat/module/elasticsearch/deprecation/test/test-json.log-expected.json

```diff
@@ -8,7 +8,7 @@
         "elasticsearch.node.name": "es1_1",
         "event.dataset": "elasticsearch.deprecation",
         "event.module": "elasticsearch",
-        "event.timezone": "+00:00",
+        "event.timezone": "-02:00",
```

Comment:
Similar to my other comment: https://github.com/elastic/beats/pull/13874/files#r330526232.